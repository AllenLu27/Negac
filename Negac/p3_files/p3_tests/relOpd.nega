void main(){
	bool res;
	res = false < false;
}
