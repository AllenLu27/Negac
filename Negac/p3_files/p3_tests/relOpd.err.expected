FATAL [3,8]: Relational operator applied to non-numeric operand
FATAL [3,16]: Relational operator applied to non-numeric operand
Semantic analysis failed
